fetools-app/src/pages/UnitConverter.jsx

```diff
@@ -19,6 +19,16 @@ function UnitConverter() {
   const [editMode, setEditMode] = useState(false);
   const [cssSize, setCssSize] = useState("16px");
 
+  // State to hold the content of the contentEditable div
+  const [editableContent, setEditableContent] = useState(
+    "Lorem ipsum dolor sit amet"
+  );
+
+  // Function to handle changes in the contentEditable div
+  const handleContentChange = (e) => {
+    setEditableContent(e.currentTarget.textContent);
```

Comment:
Because the content is replaced here, the cursor is being sent back to the beginning of the line. We need to make sure the cursor stays at the end of the line here.

This might help: https://alvarotrigo.com/blog/move-cursor-to-end-input/